source/administration/devops-command-center.rst

```diff
@@ -133,15 +133,20 @@ You can view all incidents in the incident Backstage via **Main Menu > Incidents
 - A link icon to open the corresponding incident channel
 - Status (**Ongoing** or **Ended**)
 - The commander, including profile picture and username
-- A prompt to `export the incident channel <#exporting-channels>`_
+- A prompt to `**Export the Incident Channel** <#exporting-channels>`_
 - The total duration
+  
+  - The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it shows the end time (in the user's timezone).
+
 - The number of members currently involved in the incident
+
+  - The **Members Involved** widget indicates the total number of users that participated in the channel, either
+by posting a message, being assigned as commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
```

Comment:
This doesn't need to be in a separate line

Reply:
The formatting was a bit off, so I reformatted and it looks better.

Reply:
In the bullet point for ``- The **Members Involved** widget indicates the total number of users that participated in the channel, either by posting a message, being assigned as commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.``, the paragraph is broken into a separate line at ``by posting a message...`` so the formatting is still a little off.

Reply:
That's really annoying. I'll pull this locally and fix it, thanks @amyblais.

Reply:
Great!